src/plugins/filter/filter.js

```diff
@@ -83,7 +86,8 @@ var componentName = "wb-filter",
 
 			totalEntries = $elm.find( ( settings.section || "" ) + " " + settings.selector ).length;
 
-			filterUI = "<div class=\"input-group\"><label for=\"" + inptId + "\" class=\"input-group-addon\"><span class=\"glyphicon glyphicon-filter\" aria-hidden=\"true\"></span> " + i18nText.filter_label + "</label><input id=\"" + inptId + "\" class=\"form-control " + inputClass + "\" data-" + dtNameFltrArea + "=\"" + elm.id + "\" type=\"search\"></div>" + "<p aria-live=\"polite\" id=\"" + elm.id + "-info\">" + infoFormater( totalEntries, totalEntries ) + "</p>";
+			filterUI = "<div " +  ( maxWidth ? "style=\"max-width:" + maxWidth + ";\"" : "" ) + "class=\"input-group\"><label for=\"" + inptId +
```

Comment:
We should avoid inline style. Use the plugin SASS

https://github.com/wet-boew/wet-boew/blob/master/src/plugins/filter/base.scss